### PR TITLE
6402 Remove Link Server

### PR DESCRIPTION
**Pull request by Sergio-Mira**
https://swrvedev.jira.com/browse/SWRVE-6402

Ptivate changes: https://github.com/Swrve/swrve-sdk/pull/719

### CODE REVIEW

File: Sdk/Talk/SwrveMessageController.h [hunk 1]
```diff
@@ -97,7 +97,7 @@ typedef void (^SwrveCustomButtonPressedCallback) (NSString* action);
 
 @property (nonatomic, retain) UIColor* backgroundColor;                                 /*!< Background color of in-app messages. */
 @property (nonatomic, retain) NSArray* campaigns;                                       /*!< List of campaigns available to the user. */
-@property (nonatomic, assign) id <SwrveMessageDelegate> showMessageDelegate;            /*!< Implement this delegate to intercept in-app messages. */
+@property (nonatomic, retain) id <SwrveMessageDelegate> showMessageDelegate;            /*!< Implement this delegate to intercept in-app messages. */
```
Comment (jmason) [inline Sdk/Talk/SwrveMessageController.h:100]: seems unrelated?

**Comment by jmason** [comment]
:rocket:

File: Sdk/Talk/SwrveMessageController.h (hunk 1 above)
Comment (Sergio-Mira) [inline Sdk/Talk/SwrveMessageController.h:100]: Test started failing because when using assign the instance may be released before the callback is called
